app/Main.hs

```diff
@@ -30,7 +31,7 @@ main = do
     TokenCmd TokenCredentials {..} -> do
       session <- createSession (getMatrixServer matrixServer) matrixToken
       matrixMain session xdgCache
-    CLI -> cliMain
+    CLI -> cliMain xdgCache
```

Comment:
We probably don't have enough config yet to warrant this change, but we might want to eventually have an app-specific monad in which "global" config like this is accessible via `MonadReader`, so that we don't have to pass it around.

Reply:
Yeah I agree once the executable gets a bit more sophisticated.